modules/persistent-storage-csi-gcp-file-install.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_csi-google_cloud_file.adoc
+
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-olm-operator-install_{context}"]
+= Installing the Google Filestore CSI Driver Operator
+
+The Google Filestore Container Storage Interface (CSI) Driver Operator is not installed in {product-title} by default. Use the following procedure to install the Google Filestore CSI Driver Operator in your cluster.
+
+.Prerequisites
+* Access to the {product-title} web console.
+
+.Procedure
+To install the Google Filestore CSI Driver Operator from the web console:
+
+. Log in to the web console.
+
+. Enable the Filestore API in the GCE project by using the following command:
++
+[source, command]
+----
+$ gcloud services enable file.googleapis.com  --project "my_gce_project"
+----
++
+You can also do this using Google Cloud web console.
+
+. Click *Operators* -> *OperatorHub*.
+
+. Locate the Google Filestore CSI Operator by typing *CGP Filestore CSI Driver Operator* in the filter box.
```

Comment:
Is there a typo?
Seems we need to type "GCP Filestore" in the filter box

Reply:
Done.